### PR TITLE
add two docs about clinic user guide for operator

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/clinic-user-guide-for-operator.md [hunk 1]
```diff
@@ -0,0 +1,718 @@
+---
+title: 在 Operator 部署环境使用 PingCAP Clinic 诊断服务
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:2]:
```suggestion
title: 使用 PingCAP Clinic
```

File: zh/clinic-user-guide-for-operator.md [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+summary: 详细介绍在使用 TiDB Operator 部署的集群上如何通过 PingCAP Clinic 诊断服务进行数据采集和快速检查。
+---
+
+# 在 Operator 部署环境使用 PingCAP Clinic 诊断服务
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:6]:
```suggestion
# 使用 PingCAP Clinic
```

File: zh/clinic-user-guide-for-operator.md [hunk 3]
```diff
@@ -0,0 +1,718 @@
...
+```shell
+helm search repo diag
+NAME          CHART VERSION  APP VERSION  DESCRIPTION
+pingcap/diag  v0.7.1         v0.7.1       Clinic Diag Helm chart for Kubernetes
+```
+
+#### 检查部署用户的权限
+
+部署 Diag 所使用的用户需要具备创建以下类型 *Role* 和 *Cluster Role* 的权限：
+*Role* 权限：
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:52]:
```suggestion
*Role* 权限：

```

File: zh/clinic-user-guide-for-operator.md [hunk 4]
```diff
@@ -0,0 +1,718 @@
...
+  rolebindings.rbac.authorization.k8s.io  []                 []              [get create delete]
+  roles.rbac.authorization.k8s.io         []                 []              [get create delete]
+  secrets                                 []                 []              [get list create delete]
+  services                                []                 []              [get list create delete]
+  pods                                    []                 []              [get list]
+  tidbclusters.pingcap.com                []                 []              [get list]
+  tidbmonitors.pingcap.com                []                 []              [get list]
+```
+
+*Cluster Role* 权限：
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:68]:
```suggestion
*Cluster Role* 权限：

```

File: zh/clinic-user-guide-for-operator.md [hunk 5]
```diff
@@ -0,0 +1,718 @@
...
+  tidbclusters.pingcap.com                       []                 []              [get list]
+  tidbmonitors.pingcap.com                       []                 []              [get list]
+```
+
+> **注意：**
+>
+> - 如果集群情况可以满足最小权限部署的条件，可以使用更小的权限。详情见[最小权限部署](#第-3-步部署-clinic-diag-pod)。
+
+可以通过以下步骤检查部署用户的权限：
+（1）查看部署用户绑定的 Role 角色 和 clusterRole 角色：
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:87]:
```suggestion
（1）查看部署用户绑定的 Role 角色 和 clusterRole 角色：

```

File: zh/clinic-user-guide-for-operator.md [hunk 6]
```diff
@@ -0,0 +1,718 @@
...
+```shell
+kubectl describe rolebinding -n ${namespace} | grep ${user_name} -A 7
+kubectl describe clusterrolebinding -n ${namespace} | grep ${user_name} -A 7
+```
+
+（2）查看对应角色具有的权限
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:93]:
```suggestion
（2）查看对应角色具有的权限

```

**Review by lilyjazz — APPROVED** [review]

**Comment by ran-huang** [comment]
@qqqdan do you think we need an engineer of TiDB Operator to take a look at this?

File: zh/clinic-data-instruction-for-operator.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+---
+title: Clinic 数据采集说明
```
Comment (Oreoxmt) [inline zh/clinic-data-instruction-for-operator.md:2]:
```suggestion
title: PingCAP Clinic 数据采集说明
```

File: zh/clinic-data-instruction-for-operator.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+summary: 详细说明 PingCAP Clinic 诊断服务在使用 Operator 部署的 TiDB 集群中能够采集的诊断数据类型、输出文件及采集参数。
+---
+
+# Clinic 数据采集说明
```
Comment (Oreoxmt) [inline zh/clinic-data-instruction-for-operator.md:6]:
```suggestion
# PingCAP Clinic 数据采集说明
```

File: zh/TOC.md [hunk 9]
```diff
@@ -36,6 +36,8 @@
   - [跨多个 Kubernetes 集群监控 TiDB 集群](deploy-tidb-monitor-across-multiple-kubernetes.md)
   - [开启 TidbMonitor 动态配置](enable-monitor-dynamic-configuration.md)
   - [开启 TidbMonitor 分片功能](enable-monitor-shards.md)
+  - [使用 PingCAP Clinic](clinic-user-guide-for-operator.md)
+  - [Clinic 数据采集范围说明](clinic-data-instruction-for-operator.md)
```
Comment (Oreoxmt) [inline zh/TOC.md:40]:
```suggestion
 - [使用 PingCAP Clinic](clinic-user-guide-for-operator.md)
 - [Clinic 数据采集范围说明](clinic-data-instruction-for-operator.md)
```

**Comment by ran-huang** [comment]
> [CLA assistant check](https://cla-assistant.io/pingcap/docs-tidb-operator?pullRequest=1819) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/pingcap/docs-tidb-operator?pullRequest=1819) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ Oreoxmt❌ qqqdanYou have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/pingcap/docs-tidb-operator?pullRequest=1819) it.

@qqqdan Please sign CLA

File: zh/clinic-user-guide-for-operator.md [hunk 10]
```diff
@@ -0,0 +1,724 @@
...
+```bash
+curl -s http://${host}:${port}/api/v1/data/${id}/upload -XPOST
+{
+        "date": "2021-12-10T11:26:39Z",
+        "id": "fMcXDZ4hNzs",
+        "status": "accepted"
+}
+```
+
+返回命令结果只代表上传任务开始已经开始，并不表示已完成上传。要了解上传任务是否完成，需要通过下一步操作来查看任务状态。
```
Comment (ran-huang) [inline zh/clinic-user-guide-for-operator.md:590]:
```suggestion
返回命令结果只代表上传任务已经开始，并不表示已完成上传。要了解上传任务是否完成，需要通过下一步操作来查看任务状态。
```

File: zh/clinic-user-guide-for-operator.md [hunk 11]
```diff
@@ -0,0 +1,724 @@
...
+    PdConfig_172.20.14.102:2379   PdConfig.log.file.max-backups:0   warning
+    PdConfig_172.20.15.222:2379   PdConfig.log.file.max-backups:0   warning
+
+    Result report and record are saved at /diag-fPrz0RnDxRn/report-220208030210
+    ```
+
+    上述示例中：
+
+    - 第一部分为诊断集群名称等基础信息。
+    - 第二部分为断数据来源信息。
```
Comment (ran-huang) [inline zh/clinic-user-guide-for-operator.md:722]:
```suggestion
    - 第二部分为诊断数据来源信息。
```

File: zh/TOC.md (hunk 9 above)
Comment (ran-huang) [inline zh/TOC.md:40]: 1. 建议修改文件名如下：
```suggestion
  - [使用 PingCAP Clinic 诊断 TiDB 集群](clinic-user-guide.md)
  - [Clinic 数据采集范围说明](clinic-data-collection.md)
```
2. 《Clinic 数据采集范围说明》建议放到 **参考** 下（例如与 **日志收集**、**Kubernetes 监控与告警** 平级）

File: zh/clinic-user-guide-for-operator.md [hunk 12]
```diff
@@ -0,0 +1,724 @@
...
+- 当集群出现问题，要咨询 PingCAP 技术支持时，需要提供集群诊断数据，协助技术支持人员定位问题。
+- 保留集群诊断数据，进行后期分析。
+
+> **注意：**
+>
+> 对于使用 TiDB Operator 部署的集群，暂不支持收集日志、配置文件、系统硬件信息等诊断数据。
+
+### 第 1 步：确定需要采集的数据
+
+如需查看 Clinic Diag 支持采集的数据详细列表，请参阅 [Clinic 数据采集说明 - Operator 环境](https://clinic-docs.vercel.app/docs/getting-started/clinic-data-instruction-for-operator)。建议采集完整的监控数据，以便提升诊断效率。
```
Comment (Oreoxmt) [inline zh/clinic-user-guide-for-operator.md:445]:
```suggestion
如需查看 Clinic Diag 支持采集的数据详细列表，请参阅 [Clinic 数据采集说明 - Operator 环境](https://clinic-docs.vercel.app/docs/getting-started/data-instruction/operator)。建议采集完整的监控数据，以便提升诊断效率。
```

File: zh/clinic-user-guide.md [hunk 13]
```diff
@@ -0,0 +1,722 @@
...
+- 当集群出现问题，要咨询 PingCAP 技术支持时，需要提供集群诊断数据，协助技术支持人员定位问题。
+- 保留集群诊断数据，进行后期分析。
+
+> **注意：**
+>
+> 对于使用 TiDB Operator 部署的集群，暂不支持收集日志、配置文件、系统硬件信息等诊断数据。
+
+### 第 1 步：确定需要采集的数据
+
+如需查看 Clinic Diag 支持采集的数据详细列表，请参阅 [Clinic 数据采集说明 - Operator 环境](https://clinic-docs.vercel.app/docs/getting-started/data-instruction/operator)。建议采集完整的监控数据，以便提升诊断效率。
```
Comment (ran-huang) [inline zh/clinic-user-guide.md:443]:
```suggestion
如需查看 Clinic Diag 支持采集的数据详细列表，请参阅 [Clinic 数据采集说明](clinic-data-instruction.md)。建议采集完整的监控数据，以便提升诊断效率。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: zh/clinic-user-guide.md [hunk 14]
```diff
@@ -0,0 +1,722 @@
...
+
+#### 注册并登录 Clinic Server
+
+登录 [Clinic Server](https://clinic.pingcap.com.cn/portal/#/login)，选择 **Sign in with AskTUG**，可以通过 TiDB 社区帐号登录 PingCAP Clinic 服务。若你还没有 TiDB 社区帐号，可以在登录界面进行注册。
+
+#### 创建组织
+
+用户第一次登录成功后，需要创建组织。根据页面提示输入组织名称，即可创建。创建成功后进入组织页面，可以获取 Token 后通过 Diag 的命令行或接口上传。
+
+#### 获取客户端上传 Token
```
Comment (TomShawn) [inline zh/clinic-user-guide.md:117]: 这些标题是“第 2 步：登录 Clinic Server 获取 Access Token”的子步骤吧？如果不用序号，可能不太能体现出各标题间的先后顺序关系，让人以为是平行的。能否改为步骤格式？1.  2. 3.?

Comment (Oreoxmt) [reply zh/clinic-user-guide.md:117]: > 这些标题是“第 2 步：登录 Clinic Server 获取 Access Token”的子步骤吧？如果不用序号，可能不太能体现出各标题间的先后顺序关系，让人以为是平行的。能否改为步骤格式？1. 2. 3.?

https://github.com/pingcap/docs-tidb-operator/pull/1900

File: zh/clinic-user-guide.md [hunk 15]
```diff
@@ -0,0 +1,722 @@
...
+
+对于使用 TiDB Operator 部署的 TiDB 集群，PingCAP Clinic 诊断服务（以下简称为 PingCAP Clinic）可以通过 Diag 诊断客户端（以下简称为 Diag）与 Clinic Server 云诊断平台（以下简称为 Clinic Server）实现远程定位集群问题和本地快速检查集群状态。
+
+> **注意：**
+>
+> 本文档**仅**适用于使用 TiDB Operator 部署的集群。如需查看适用于使用 TiUP 部署的集群，请参阅 [TiUP 环境的 Clinic 操作手册](https://docs.pingcap.com/zh/tidb/dev/clinic-user-guide-for-tiup)。
+>
+> PingCAP Clinic 暂时**不支持**对 TiDB Ansible 部署的集群进行数据采集。
+
+对于使用 TiDB Operator 部署的集群，Diag 需要部署为一个独立的 Pod。本文介绍如何使用 `kubectl` 命令创建并部署 Diag Pod 后，通过 API 调用继续数据采集和快速检查。
```
Comment (TomShawn) [inline zh/clinic-user-guide.md:16]: “继续”，是说在这之前有数据采集和快速检查吗？

Comment (Oreoxmt) [reply zh/clinic-user-guide.md:16]: > “继续”，是说在这之前有数据采集和快速检查吗？

应该是“通过调用 API 进行数据采集和快速检查”

**Pull request merged**